src/main/java/frc/robot/RobotContainer.java

```diff
@@ -22,80 +38,210 @@
+   * edu.wpi.first.wpilibj2.command.button.CommandGenericHID}'s subclasses for
+   * {@link
+   * CommandXboxController
+   * Xbox}/{@link edu.wpi.first.wpilibj2.command.button.CommandPS4Controller
+   * PS4} controllers or
+   * {@link edu.wpi.first.wpilibj2.command.button.CommandJoystick Flight
    * joysticks}.
    */
   private void configureBindings() {
-    
+
     m_Drive.setDefaultCommand(Commands.run(() -> {
-      double forwardSpeed = -contrJoystick.getY();
-      double rightSpeed = -contrJoystick.getX();
-      double rotatinalSpeed = -contrJoystick.getZ();
-      m_Drive.drive(forwardSpeed, rightSpeed, rotatinalSpeed);
+      double forwardSpeed = -driveJoystick.getY();
+      double rightSpeed = driveJoystick.getX();
+      double rotatinalSpeed = driveJoystick.getZ();
+      double slider = driveJoystick.getRawAxis(3);
+      m_Drive.drive(forwardSpeed, rightSpeed, rotatinalSpeed, slider);
     }, m_Drive));
 
-    contrJoystick.button(1).onFalse(Commands.run(() -> {
-      m_LaunchMech.launchSpeaker();
-    }, m_LaunchMech));
-
-    contrJoystick.button(2).onFalse(Commands.run(() -> {
-      m_LaunchMech.launchAmp();
-    },m_LaunchMech));
-
-    contrJoystick.button(3).whileTrue(Commands.run(() -> {
-      m_Intakemech.intake();
-    },m_Intakemech));
-
-    contrJoystick.button(3).onFalse(Commands.run(() -> {
-      m_TrapScoreSubsystem.startAngle();
-    },m_TrapScoreSubsystem));
+    m_Navigation.setDefaultCommand(Commands.run(() -> {
+      m_Navigation.ledsOff();
+    }, m_Navigation));
+
+    driveJoystick.button(12).onTrue(Commands.runOnce(() -> m_Navigation.resetPosition(), m_Navigation));
+    driveJoystick.button(11).onTrue(Commands.runOnce(() -> m_Navigation.resetFieldOrient(), m_Navigation));
+    driveJoystick.button(7).whileTrue(Commands.run(() -> m_Drive.alignSpeaker(), m_Drive));
+    driveJoystick.button(8).whileTrue(Commands.run(() -> m_Drive.alignAmp(), m_Drive));
+    driveJoystick.button(10).onTrue(Commands.run(() -> m_Navigation.ledsOn(), m_Navigation));
+    driveJoystick.button(9).onTrue(Commands.runOnce(() -> m_Drive.toggleFieldOrient()));
+
+    driveJoystick.button(5).whileTrue(Commands.run(() -> mLeftClimber.raise(), mLeftClimber));
+    driveJoystick.button(3).whileTrue(Commands.run(() -> mLeftClimber.lower(), mLeftClimber));
+    driveJoystick.button(3).onFalse(Commands.runOnce(() -> mLeftClimber.stop(), mLeftClimber));
+    driveJoystick.button(5).onFalse(Commands.runOnce(() -> mLeftClimber.stop(), mLeftClimber));
+
+
+    driveJoystick.button(6).whileTrue(Commands.run(() -> mRightClimber.raise(), mLeftClimber));
+    driveJoystick.button(4).whileTrue(Commands.run(() -> mRightClimber.lower(), mLeftClimber));
+    driveJoystick.button(4).onFalse(Commands.runOnce(() -> mRightClimber.stop(), mLeftClimber));
+    driveJoystick.button(6).onFalse(Commands.runOnce(() -> mRightClimber.stop(), mLeftClimber));
+
+
+    //driveJoystick.button(1).onTrue(Commands.run(() -> m_Drive.sportMode(true)));
```

Comment:
We should be removing commented code before commiting. Goes for me as well.

Reply:
Reviewed everything. Removed all of the commented code.